### DISCUSSION TITLE
增加原书目录查询

### DISCUSSION THREAD

**Discussion by ZySieng** [MindongLab/yngdieng#67] [产品功能 ideas]
**你提出的功能想解决什么问题**
目前榕典的高级搜索，实质上是针对单字音的音韵搜索。
可以添加原书目录搜索。

**你觉得理想的解决方案是什么**
举冯书为例，p1-19是i韵；20-22是ei韵；23-38是u韵……
在目录查询中，我们提供按原书目录顺序的韵母表格，

i
ei
u
ou

点击“i”后，显示条目按原书顺序排列。

【捭】……
【捭鬮】……
【碑】……

如此来实现类似翻阅冯书原书的效果。
————————
除了目录搜索之外，还可以提供分类词汇查询，冯书P443开始，就是分类词汇，输入法录入时也是按这个顺序来的。
按照目录查询的思路，分类词汇查询显示如下：

壹 天文
（1）日、月、星
（2）風、雲、雷、雨

点击“（1）日、月、星”后，显示条目

【日頭】……
【半爿日】……
【照照日】……

————————
我们现在的高级搜索，我觉得可以变为原书目录搜索。平时使用的，是普通搜索，可以检索所有词库。而原书搜索，是针对特定书目，使用原书检索方式的、数位化的查询。
现在的高级搜索实质上也是一种原书搜索，是按照八音编排的顺序，按声韵调来搜索单字的，实际上也就是《戚林八音校注》的检索方式（《榕注》因为大黑录入的是八音音系，虽然原书检索方式不同于八音，但是在我们这里依然是使用八音去记音与检索的）

**其他可能的解决方案**
你考虑过的，能解决同样问题的其他可能方案。

**其他信息**

**Comment by ZySieng** [comment]
这个问题我觉得优先级要设置得高一些，放在林圜配音前后，特别是冯书的分类词汇，很重要。@ztl8702 
因为榕典给我的感觉是，里面虽然有很多数位化的文本，但是查询方式比较低效。举例如下：
1. 有些词汇记载在原书里面，我不知道它的发音、不知道它的汉字也不知道普通话释义时，用目前的查询方式等于是根本找不到它。
2. 面对不知音、写法，但想要通过普通话了解一个对应意义的福州话词汇时，也不太好用，因为只能检索到原书注释里面的普通话文本（这涉及另外一个功能，我等等发issue）。

除此之外，拼音检索的模糊音优化与支持异体字也要考虑，其中前者比后者重要，但是二者都没有原书目录查询重要。

**Comment by ztl8702** [comment]
可以新设一个“浏览”功能。UI设计我近期制作。
